thirdparty/README.md

```diff
@@ -665,6 +665,17 @@ They can be reapplied using the patches included in the `vhacd`
 folder.
 
 
+## volk
+
+- Upstream: https://github.com/zeux/volk
+- Version: 1.2.170.0 (b18f3192886981f28d17f8f41c26c5a53ac3b977, 2021)
```

Comment:
I'd think with https://github.com/zeux/volk/commit/d75c007f375f35612dba3de512ac73f10bf9aa0e, `1.2.182` is the latest SDK release in the Vulkan stack: https://github.com/KhronosGroup/Vulkan-Loader/releases/tag/sdk-1.2.182.0

Reply:
Updated `headers` and `volk` and `vk_enum_string_helper` to the 1.2.182.